### DISCUSSION TITLE
@types/jest has no version 27

### DISCUSSION THREAD

**Discussion by keeganwitt** [DefinitelyTyped/DefinitelyTyped#54606] [Issues with a @types package] [answered]
I apologize if this isn't the appropriate venue for this, but I wasn't sure how else to get in contact with the maintainers. Jest documentation [states](https://jestjs.io/docs/getting-started#using-typescript)

> For @types/* modules it's recommended to try to match the version of the associated module. For example, if you are using 26.4.0 of jest then using 26.4.x of @types/jest is ideal. In general, try to match the major (26) and minor (4) version as closely as possible.

But there is no version 27 of `@types/jest`. Is this because there are no breaking API changes? Comparing https://jestjs.io/docs/api and https://jestjs.io/docs/26.x/api, I didn't notice any offhand. If so, then it's theoretically possible that the package would jump from 26 to 28 if that's when breaking changes are introduced? I thought this was the only time there hadn't been a matching major version released, but I guess there was one other time this was the case (17).

**Comment by domdomegg** [comment, marked as answer]
I've raised a PR to update Jest to version 27: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/55013
